internal/pkg/api/APIServer.go

```diff
@@ -18,15 +18,19 @@ package api
 
 import (
 	"github.com/gorilla/mux"
-	"github.com/sirupsen/logrus"
-	"github.com/wso2/micro-gw/internal/pkg/confTypes"
```

Comment:
Instead of using like <packagename>/internal/pkg/confTypes, why do we import it from github? Is this intentional?

Reply:
Yeah It's international. Otherwise we need to import all the packages using go gopath.(It is much difficult. go doesn't support relative paths) Here we use go modules to avoid that difficulty. Since it behaves like we import it from github.